src/Hyperion.Tests/CrossFrameworkSerializationTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.IO;
+using Hyperion.Tests.Generator;
+using Xunit;
+
+namespace Hyperion.Tests
+{
+    public class CrossFrameworkSerializationTests
+    {
+        private readonly Serializer _serializer;
+        private readonly CrossFrameworkClass _originalObject;
+
+        public CrossFrameworkSerializationTests()
+        {
+            _serializer = new Serializer();
+            _originalObject = CrossFrameworkInitializer.Init();
+        }
+
+        [Fact]
+        public void CanSerializeCrossFramework()
```

Comment:
Nice, this the only way to do it - have to actually load the sucker and work the payloads on both ends.